Fix screen capture example

wbamberg opened the pull request:
This example was broken because of assigning to `const logger`. With that fixed the logging code still didn't actually log, so this fixes that too.

This PR also simplifies the logger and removes some "simple", "easy" etc. Nails on a chalkboard.

The example still doesn't work for me on Firefox but I think this might be a permissions thing on my end?

wbamberg committed:
Fix screen capture example

teoli2003 reviewed (changes requested):
Two questions…

teoli2003 commented on files/en-us/web/api/screen_capture_api/using_screen_capture/index.md:
```diff
@@ -298,33 +297,27 @@ The key parts of the HTML are:
 The final product looks like this. If your browser supports Screen Capture API, clicking "Start Capture" will present the {{Glossary("user agent", "user agent's")}} interface for selecting a screen, window, or tab to share.
 
-{{EmbedLiveSample("Simple_screen_capture", 640, 680, "", "", "", "display-capture")}}
+{{EmbedLiveSample("Streaming screen capture", 640, 760, "", "", "", "display-capture")}}
```
An extra question: what does the `display-capture` parameter do?

bsmth replied:
Works okay for me on Fx stable & nightly although the logger CSS looks a little strange:

<img width="937" alt="image" src="https://user-images.githubusercontent.com/43580235/224046097-63bdd6ee-8ead-4bb2-b256-85c9e3f9481f.png">

wbamberg committed:
Fix various problems

wbamberg commented:
I hope it is less broken now!

- removed the `logger` variable
- fixed the selector for the logger
- hopefully fixed the log size and overflow so it won't do ^that

teoli2003 reviewed (approved):
Looks good to me and works well on Firefox Nightly.
